### PR TITLE
fix: add gql transformer version to status output

### PR DESCRIPTION
**Pull request by cjihrig**
Fixes: https://github.com/aws-amplify/amplify-cli/issues/9160

### CODE REVIEW

**Review by danielleadams — CHANGES_REQUESTED** [review]
Would it be possible to add a unit test for showGraphQLTransformerVersion?

**Comment by cjihrig** [comment]
@danielleadams tests added. Please take a look.

**Review by danielleadams — APPROVED** [review]

File: packages/amplify-provider-awscloudformation/src/display-helpful-urls.js [hunk 1]
```diff
@@ -278,7 +280,24 @@ function showGraphQLTransformerMigrationMessage() {
...
+    .filter(([_, apiResource]) => apiResource.service === 'AppSync')
+    .map(([name]) => name).length;
+
+  if (!hasGraphqlApi) {
+    return;
+  }
+
+  const transformerVersion = await getTransformerVersion(context);
+
+  context.print.info(chalk`GraphQL transformer version: ${transformerVersion}`);
```
Comment (jhockett) [inline packages/amplify-provider-awscloudformation/src/display-helpful-urls.js:296]: nit: use `printer` from amplify-prompts

`chalk` without `()` looks like a typo possibly?

```js
chalk`GraphQL transformer version: ${transformerVersion}`
```

Comment (cjihrig) [reply packages/amplify-provider-awscloudformation/src/display-helpful-urls.js:296]: > nit: use printer from amplify-prompts

I originally did, but changed it for easier mocking for the requested unit tests.

> chalk without () looks like a typo possibly?

That is the style used throughout that file.

Comment (jhockett) [reply packages/amplify-provider-awscloudformation/src/display-helpful-urls.js:296]: TIL that is valid JS.

**Pull request merged**